src/main/java/seedu/duke/Parser.java

```diff
@@ -55,134 +216,59 @@ static void setProjectIndex(int newIndex) {
      */
     public static Command checkAction(String inputCommand) throws DukeExceptions {
         Command commandType = null;
-        String[] inputs = inputCommand.split("\\s+");
+        String[] inputs = inputCommand.split("\\s+", 2);
         String taskType = inputs[0];
-        String description = "";
+
+        HashMap<String, String> params = new HashMap<>();
+        if (inputs.length == 2) {
+            String paramsString = inputs[1];
```

Comment:
Sean just one question over here ah, we will only able to accept one param over here issit?  something like "select t/2". But will this block of code be able to accept a command like "deadline t/2 d/2020-10-18" where there are two params ?

Reply:
I think the '2' is the limit for integers so in that sense I believe Shreyas' concern is valid.
I have 2 suggestions:
- We loop through the inputs and extract out key-map info for multiple params
- We only include 1 param type for all our commands

Reply:
No it can accept more than one parameter. The split here is to separate commandType from the parameters. Inputs[1] contains the whole string beside the first word.

Reply:
As of now, the function already loops through and extracts all params.

Reply:
Ok got it thanks for the clarification